mautic/mautic
# Fix owner tokens to store to email stats

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
At the moment on each email sent are owner tokens stored in email_stats table, even If we do not use it. 
This PR fixed it
With mills of rows, It could save some space on the disk. 



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Create email with `{ownerfield=firstname}` token
3. Send email and check `email_stat` table `tokens` column
4. You see all `ownerfield  tokens are stored there
5. After PR you should see  just `firstname` token
